### ISSUE TITLE
Column default value not work

### ISSUE THREAD

**Issue by MEiDIK** [requery/requery#579]
Thanks for this great lib~

i have defined the user bean as: 

```
@Table(name = "User")
@Entity(model = "global")
interface IUser : Persistable {

    @get:Key @get:Generated var id: Long

    @get:Column(value = "'Newer'")
    var name: String?

    var avatar: String?

}
```
when i use the follow code to insert a new User into the db, the name column in the db is empty..

```
db.insert(User().apply {
                    this.avatar = "http://......"
                })
```

it sense like insert a "null" name into the db and cover the default value

how can i fix it?

**Comment by npurushe** [comment]
This may be related to #505 basically the null value is not ignored on insert. The create table should be defined to have name with a default value though.

**Comment by npurushe** [comment]
Fixed by change for #505

**Comment by MEiDIK** [comment]
Thanks～～
